app/src/test/java/org/simple/clinic/summary/linkId/LinkIdWithPatientSheetControllerTest.kt

```diff
@@ -0,0 +1,72 @@
+package org.simple.clinic.summary.linkId
+
+import com.nhaarman.mockito_kotlin.any
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.never
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.whenever
+import io.reactivex.Single
+import io.reactivex.subjects.PublishSubject
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.patient.PatientRepository
+import org.simple.clinic.patient.businessid.Identifier
+import org.simple.clinic.util.RxErrorsRule
+import org.simple.clinic.widgets.UiEvent
+import java.util.UUID
+
+class LinkIdWithPatientSheetControllerTest {
+
+  @get:Rule
+  val rxErrorsRule = RxErrorsRule()
+
+  private val patientRepository = mock<PatientRepository>()
+
+  private val sheet = mock<LinkIdWithPatientSheet>()
+
+  private val uiEvents = PublishSubject.create<UiEvent>()
+
+  private lateinit var controller: LinkIdWithPatientSheetController
+
+  private val patientUuid = UUID.randomUUID()
+
+  private lateinit var identifier: Identifier
```

Comment:
Similarly here, identifier can be turned into `val`.

Reply:
I will do this.